_episodes/04-formatting.md

```diff
@@ -222,6 +222,68 @@ RETURN (0)
 ~~~
 {: .language-sql}
 
+> ## Highlighting for other languages
+> You may use other `language-*` classes to activate syntax highlighting
+> for other languages.
+> For example,
+>
+> {% raw %}
+>     ~~~
+>     title: "YAML Highlighting Example"
+>     description: "This is an example of syntax highlighting for YAML."
+>     array_values:
+>         - value_1
+>         - value_2
+>     ~~~
+>     {: .language-yaml }
+> {% endraw %}
+>
+>
+> will produce this:
+>
+> ~~~
+> title: "YAML Highlighting Example"
+> description: "This is an example of syntax highlighting for YAML."
+> array_values:
+>     - value_1
+>     - value_2
+> ~~~
+> {: .language-yaml }
+>
+> and combining `.language-*` with the `.source` class with add the generic
+> "Code" title to the block.
+>
+> {% raw %}
+>     ~~~
+>     for i = 1:2, j = 3:4
+>         println((i, j))
+>         i = 0
+>     end
+>     ~~~
+>     {: .source .language-julia }
+> {% endraw %}
+>
+> ~~~
+> for i = 1:2, j = 3:4
+>     println((i, j))
+>     i = 0
+> end
+> ~~~
+> {: .source .language-julia }
```

Comment:
This is currently highlighted as the default 'Code' block.

Reply:
Oh, it actually makes sense because you use `.source` to get nice border around this block